django_filters/rest_framework/backends.py

```diff
@@ -93,5 +93,5 @@ def get_schema_fields(self, view):
                     description=six.text_type(field.field.help_text)
                 )
             )
-            for field_name, field in filter_class().filters.items()
+            for field_name, field in filter_class(request=view.request).filters.items()
```

Comment:
Changed. This attribute always exists for schema - https://github.com/encode/django-rest-framework/blob/master/rest_framework/schemas.py#L391